Element/Digitizer.php

```diff
@@ -242,7 +242,9 @@ public function httpAction($action)
                 if (empty($request['dataItem'][ $uniqueIdKey ])) {
                     unset($request['dataItem'][ $uniqueIdKey ]);
                 }
-                $results = $dataStore->save($dataItem);
+
+                $dataIems = $dataStore->save($dataItem);
```

Comment:
Is this a typo? `$dataIems` should be `$dataItems`?

In addition is it not better to combine both lines into one like this:

```php
$results = $dataStore->save($dataItem)->toArray();
```